lecture02.tex

```diff
@@ -25,6 +25,14 @@ \section{Lecture 2}
 Recall that the algebra $B$ is said to be \emph{of finite type} provided that $B$ is finitely generated, meaning that $B = A[b_{1},\ldots,b_{n}]$ for some choice of $b_{1},\ldots,b_{n} \in B$.
 A stronger notion of finiteness is being a \emph{finite} algebra, which means that $B$ is finitely generates as an $A$-module.
 
+\begin{lemma}[Transitivity of Finiteness]\label{lemma:transitivity of finiteness}
+	Let $A$ be a finitely generated $B$-module and $B$ a finitely generated $C$-module. Then $A$ is finitely generated as a $C$-module. 
+\end{lemma}
```

Comment:
Please formulate the lemmas as follows.
```suggestion
\begin{lemma}[Transitivity of Finiteness]\label{lemma:transitivity of finiteness}
  Let $B$ be a finite $A$-algebra and let $C$ be a finitely-generated $B$-module.
  Then $C$ is a finitely-generated $A$-module.
\end{lemma}
```
Also, please use 2 spaces instead of tabs, and ensure that each sentence is on it's own line.